app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -88,6 +129,46 @@ class StudentForm extends React.Component {
       reject
     );
   }
+  
+  renderSchoolLevelOptions() {
+    var options = [];
+    for (var i = 0; i < schoolLevels.length; i++) {
+      options.push(<option value={i}>{schoolLevels[i]}</option>);
+    }
+    return options;
+  }
```

Comment:
is there a cleaner way to do all the renderBlankOptions()??

For example if you defined a renderOptions(array) method that took in the array to render that would remove a lot of duplicate code